proto/internal/temporal/server/api/deployment/v1/message.proto

```diff
@@ -182,6 +189,24 @@ message SyncVersionStateResponse {
     VersionLocalState version_state = 1;
 }
 
+// Sent from Version workflow to Worker Deployment workflow
+message AddVersionUpdateArgs {
+    string version = 1;
+    google.protobuf.Timestamp create_time = 2;
+}
+
+// Sent from Drainage child workflow to Version parent
+message SyncDrainageInfoSignalArgs {
+    temporal.api.deployment.v1.VersionDrainageInfo drainage_info = 1;
+}
+
+// Sent from Version workflow to Worker Deployment workflow
+message SyncDrainageStatusSignalArgs {
+    string version = 1;
+    temporal.api.enums.v1.VersionDrainageStatus drainage_status = 2;
+}
```

Comment:
these are not currently used, I wanted to keep this PR short, but I will have another PR using them so that we can keep the VersionSummaries up to date in the deployment WF